src/app/directives/item-tooltip.directive.ts

```diff
@@ -20,6 +20,10 @@ export class ItemTooltipDirective {
     cdkDrag.started.pipe(untilDestroyed(this)).subscribe(_ => this.destory());
   }
 
+  ngOnDestroy(): void {
```

Comment:
why this?

Reply:
To remove the tooltip when the item is deleted. Otherwise the tooltip stays when hovering and item and then deleting it.